README.md

```diff
@@ -73,7 +73,11 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
 
 The website is deployed using `now.sh`. The production website reflects the `master` branch. Pull requests which are made against master are considered staging branches.
 
-When contributing to the project branch from master and create a pull request, make sure that changes are approved and that your staging branch works as you expect. **When you merge into master the change will shortly be available on the production site.**
+1. When contributing to the project branch from master and create a pull request.
```

Comment:
Shouldn't this be "creating a pull request" to match the present participle "contributing"?